voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
@mehmetalisavas that sounds good! As far as I'm concerned only attribution is missing from this code change.

I completely agree having the block instead of `url.Port`